app/routes/circulars.$circularId.($version)/route.tsx

```diff
@@ -166,29 +171,31 @@ function CircularsHistory({
       >
         Versions
       </DetailsDropdownButton>
-      {showVersions && (
-        <DetailsDropdownContent>
-          <CardBody>
-            <Link
-              onClick={() => setShowVersions(!showVersions)}
-              to={`/circulars/${circular}`}
-            >
-              Latest
-            </Link>
-            {history &&
-              history.map((version) => (
-                <div key={version}>
-                  <Link
-                    onClick={() => setShowVersions(!showVersions)}
-                    to={`/circulars/${circular}/${version}`}
-                  >
-                    Version {version}
-                  </Link>
-                </div>
-              ))}
-          </CardBody>
-        </DetailsDropdownContent>
-      )}
-    </>
+      <DetailsDropdownContent
+        className={classNames('maxw-card-xlg', {
+          'display-none': !showVersions,
+        })}
+      >
```

Comment:
Is it important that we switch from conditionally rendering this content to conditionally styling it as not displayed?

Reply:
Yes, please switch it back.

Reply:
reverted in [652d205](https://github.com/nasa-gcn/gcn.nasa.gov/pull/2362/commits/652d2057aab6cb8169e82628ceb3e9a9edc7b9e9)